### PR TITLE
41 data pipeline skeleton

### PR DESCRIPTION
**Pull request by DarylDohner**

### CODE REVIEW

**Comment by DarylDohner** [comment]
(forgot to add the dummy tables to the sql files, will do soon)

**Comment by DarylDohner** [comment]
(forgot to add the dummy tables to the sql files, will do soon)
^ actually I think this is not the case, I think this ticket may technically be blocked by #24 @David-Rey

File: EosGround/database/db_setup.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import os
+import psycopg2
+
+from EosGround.config.config import get_config
+
+
+def configure_db():
```
Comment (DarylDohner) [inline EosGround/database/db_setup.py:7]: Not sure why github isn't rendering the diff but the only changes I made were moving the file, making filepaths cross-platform, adding some print statements, and adding the test_schema setup

**Review by thomasmholder — CHANGES_REQUESTED** [review]
Looks pretty good, but I think you should add an example foreign key relationship to the test schemas, I got one working but getting the name right in the model was a little fiddly, and pretty much every pipeline is going to have a foreign key relationship.

File: requirements.txt [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
+exceptiongroup==1.1.0
+greenlet==2.0.2
 iniconfig==1.1.1
 packaging==21.3
 pluggy==1.0.0
 pyparsing==3.0.9
 pytest==7.2.1
 tomli==2.0.1
 digi-xbee==1.4.1
+psycopg2==2.9.5
```
Comment (thomasmholder) [inline requirements.txt:13]: We should use either `psycopg2` or `psycopg2-binary`, I see having both in our dependencies causing issues

Comment (thomasmholder) [reply requirements.txt:13]: (My vote is for `psycopg2-binary` but I can figure out how to build the other one if I have to)

**Review by thomasmholder — CHANGES_REQUESTED** [review]
Looks good and works on my end, just one thing with the dependencies

**Review by thomasmholder — APPROVED** [review]

**Comment by thomasmholder** [comment]
Lgtm

**Pull request merged**